app/src/main/java/org/schabi/newpipe/util/Constants.kt

```diff
@@ -0,0 +1,20 @@
+@file:JvmName("Constants")
+
+package org.schabi.newpipe.util
+
+/**
+ * Default duration when using throttle functions across the app, in milliseconds.
+ */
+const val DEFAULT_THROTTLE_TIMEOUT = 120L
```

Comment:
I see there are no other file changes. Isn't this variable used anywhere? If that's the case remove it

Reply:
It's used in some of the `ViewModel`s.

Reply:
Shouldn't the `ViewModel`s imports also be changed? Otherwise they would point to `ConstantsKt`, wouldn't they?

Reply:
The `ViewModel`s are written in Kotlin, so no change is needed.